src/openrct2/ride/Vehicle.cpp

```diff
@@ -9448,21 +9448,21 @@ static int32_t vehicle_update_track_motion_mini_golf(Vehicle* vehicle, int32_t*
  *
  *  rct2: 0x006DC1E4
  */
-static void vehicle_update_track_motion_powered_ride_acceleration(
-    Vehicle* vehicle, rct_ride_entry_vehicle* vehicleEntry, uint32_t totalMass, int32_t* acceleration)
+void Vehicle::UpdateTrackMotionPoweredRideAcceleration(
+    rct_ride_entry_vehicle* vehicleEntry, uint32_t totalMass, int32_t* curAcceleration)
```

Comment:
I think I'd stop sending the pointer and start returning the new acceleration value, I think the interface becomes clearer